roles/sql-command/tasks/create-user.yml

```diff
@@ -38,7 +38,7 @@
     - name: Remove pgpass file
       include: remove-pgpass.yml
 
-  when: "{{ rds_dbs | selectattr('db_engine', 'match', 'postgres')| list |count > 0 }}"
+  when: rds_dbs | selectattr('db_engine', 'match', 'postgres')| list |count > 0
 
```

Comment:
L71: is this working? , multiple jinja filters on a single line always give me upset stomachs, see https://github.com/ansible/ansible/issues/17773#issuecomment-249963023